cmd/appliance/upgrade/complete.go

```diff
@@ -102,21 +94,14 @@ func NewUpgradeCompleteCmd(f *factory.Factory) *cobra.Command {
 			return nil
 		},
 		RunE: func(c *cobra.Command, args []string) error {
-			h, err := opts.Config.GetHost()
-			if err != nil {
-				return fmt.Errorf("could not determine hostname for %s", err)
-			}
-			if err := network.ValidateHostnameUniqueness(h); err != nil {
-				return err
-			}
```

Comment:
I don't understand why this was removed, and the commit message/pr description does not describe it. how come?

Reply:
Because it's moved. It's run when identifying the primary controller instead